reana_client/cli/files.py

```diff
@@ -555,7 +555,7 @@ def workflow_disk_usage(
 ):  # noqa: D301
     """Get workspace disk usage.
 
-    The ``du`` command allows to chech the disk usage of given workspace.
+    The ``du`` command allows to check the disk usage of given workspace.
```

Comment:
Please update also `CHANGES.rst` to add something like:

```
Fixed ``du`` command help message typo.
```

Even though such a release message is perhaps not very important, we could use this to practice our review habits :wink: